src/renderer/features/settings/components/general/control-settings.tsx

```diff
@@ -170,6 +173,26 @@ export const ControlSettings = () => {
       isHidden: false,
       title: 'Volume wheel step',
     },
+    {
+      control: (
+        <Switch
+          defaultChecked={settings.resume}
+          disabled={!isElectron()}
+          onChange={(e) => {
+            localSettings?.set('resume', e.target.checked);
+            setSettings({
+              general: {
+                ...settings,
+                resume: e.currentTarget.checked,
+              },
+            });
+          }}
+        />
+      ),
+      description: 'When exiting, save the current play queue and restore it when reopening',
+      isHidden: false,
+      title: 'Save queue state on close',
```

Comment:
Should probably move the `isElectron()` over to `isHidden` rather than disabled.